buildSrc/build.gradle.kts

```diff
@@ -59,6 +59,7 @@ val jacksonVersion = "2.13.0"
 val googleAuthToolVersion = "2.1.2"
 val licenseReportVersion = "2.0"
 val grGitVersion = "3.1.1"
+val protobufPluginVersion = "0.8.17"
```

Comment:
Let's discuss this one. Either it needs a comment telling to sync this version (similar to `errorProneVersion` below), or we might not need this version at all — as (maybe) we always grab the most recent from Gradle plugin repository.